src/compressed_tensors/utils/offload.py

```diff
@@ -22,23 +37,37 @@
     "get_offloaded_device",
     "update_prefix_dict",
     "update_parameter_data",
+    "register_offload_parameter",
+    "update_offload_data",
+    "delete_offload_parameter",
+    "has_offloaded_params",
 ]
 
 
-def is_module_offloaded(module: Module) -> bool:
-    """
-    :param module: layer to check
-    :return: True if layer is offloaded from GPU, False otherwise
-    """
-    return hasattr(module, "_hf_hook") and module._hf_hook.offload
+def check_accelerate(fallback: Any):
+    def decorator(func: Callable[[Any], Any]):
+        if not _has_accelerate:
+            return lambda *args, **kwargs: fallback
```

Comment:
can we just return fallback here ?

Reply:
Can we also log out that user should install accelerate, similar to what you have above, showing a message to to run `...[accelerate]`?

Reply:
Remember that the return type of `decorator` is a `Callable[[Any], Any]`. For that reason, `decorator` must return a function, not the fallback itself.

Reply:
I'd prefer not to log anything, since the purpose is for users to be able to call functions such as `has_offloaded_params` without having to worry about whether accelerate is installed.